sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -107,6 +107,11 @@ public final class StorageBlobClient: PipelineClient {
                 UserAgentPolicy(for: StorageBlobClient.self),
                 RequestIdPolicy(),
                 AddDatePolicy(),
+                NormalizeETagPolicy(),
```

Comment:
I checked against the Java client, and it looks like these should be added at the end of the policy. Specifically the order these are inserted in the pipeline should be:

1.) (Everything else)
2.) HeadersValidationPolicy
3.) LoggingPolicy
4.) ScrubEtagPolicy

Reply:
Done.